Makefile

```diff
@@ -0,0 +1,12 @@
+
+ver:
+	@read -p "Enter a new version:" version; \
+	old_version=$(shell grep -o "\d.\d.\d" Dockerfile);  \
+	echo "$$old_version to $$version"; \
+	for i in $(shell find . -name "Dockerfile"); do sed -i "s/$$old_version/$$version/g" $$i; done;
+
+sync:
+	cat setup.sh | tee $(shell find . -name "gp-setup.sh")
```

Comment:
This might be a tiny bit easier to read:
```cp setup.sh $(shell find . -name "gp-setup.sh")```

Reply:
that doesn't work,  I don't think `cp` command can copy to multiple targets.